docs/guides/06-aggregations.md

```diff
@@ -0,0 +1,408 @@
+```
+color: blend(gray, red, @f_isBiggest)
+width: blend(10, 40, @f_isBiggest)
+```
+
+And add this new variable to the same viz:
+```
+@f_isBiggest: ($pop_max == @v_max)
+```
+
+> Notice how `blend` works here as a convenient way to apply a conditional style, where its parameters can be interpreted as `blend(FALSE_VALUE, TRUE_VALUE, CONDITION)`. See also that we named our feature-dependent variable with a `@f_` prefix.
+
+As a test, you can temporary disable (and then enable again) the current filter, to display more cities at the same time. Just add `//` before the `filter` line in the `viz` definition, like this:
+```
+// filter: ($pop_max > @g_p95)
+```
+> This is a practical way to de-activate a filter without removing it from the code.
+
+
+At this point, you have created this great visualization, using global and viewport functions:
+<div class="example-map">
+    <iframe
+        id="guides-aggregations-step-2"
+        src="/developers/carto-vl/examples/maps/guides/aggregations/step-2.html"
+        width="100%"
+        height="500"
+        frameBorder="0">
+    </iframe>
+</div>
+To explore the map at full screen size, have a look at [this link](/developers/carto-vl/examples/maps/guides/aggregations/step-2.html) and see the biggest cities in the world (Tokyo, New York, Mumbay...).
+
+
+### Resolution
+
+This is an interesting property you need to know about when you are displaying a layer with points and using CARTO backend (`carto.source.Dataset` or `carto.source.SQL`). The resolution is a value that defines an spatial aggregation for your features, and it can go from **1 pixel** (default) to **255px**.
+
+For that resolution value a regular grid is internally created, with the size in pixels you have indicated. Any feature falling in the same cell will be aggregated into a 'clustered point'. That point's position will be determined by the current [centroid](https://carto.com/help/glossary/#centroid) of the features found inside each cell (aggregation placement is always set to `centroid` mode).
+
+You can give it a try now modifying the resolution in the current viz, adding this property:
+```
+resolution: 128
+```
+> Remember, if there is no explicit value, `resolution = 1`. Common resolution values are <=16
+
+That should be enough to set the resolution and reduce the number of points, but depending on the layer it might be difficult to be perceived without further modifications. Let's add more properties to grasp the concept, removing the `filter` and changing the `width`, using this:
+```js
+const citiesViz = new carto.Viz(`
+    color: blend(gray, red, @f_isBiggest)
+    // width: blend(10, 40, @f_isBiggest)
+    width: 5 * clusterCount()
+    @g_max: globalMax($pop_max)
+    @g_avg: globalAvg($pop_max)
+    @g_p95: globalPercentile($pop_max, 95)
+    // filter: ($pop_max > @g_p95)
```

Comment:
Should we remove the commented lines here? Or do we want to keep them there for a reason?

Reply:
I left them on purpose in this guide, cause sometimes I say 'activate / deactivate' the filter... and alike. IMO in this guide it also helps a bit to follow the steps.

When that get's a little messy, like in the final step, I removed them all and just keep the ones been used